demo/runner.sh

```
@@ -0,0 +1,10 @@
+BINARIES="/home/velocitatem/CLionProjects/CellViz/cmake-build-debug/"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)